mantidproject/mantid
# Refl GUI searches ICAT Asynchronously

Currently it is working and the search table is populated but this is just a scratch way of doing things, hopefully with some review this can be made into a robust solution.

**To test**: 
- Load up Reflectometry (Polref) GUI (`Interfaces > Reflectometry > ISIS Reflectometry (Polref)`)
- Change search instrument from `INTER` to `POLREF`
- Change transfer method from `Description` to `Measurement`
- Enter investigation ID `1520249`
- Press search
- Login into ICAT data service
- Ensure that the search table is now populated with search results.

**Release notes**: 
Functionality has not changed from the user's point of view so no update needed
Fixes #14905
